test/edu/ucsb/nceas/metacat/dataone/MNodeQueryTest.java

```diff
@@ -708,6 +729,13 @@ public void testPackageWithSID() throws Exception {
         query = "q=id:"+resourceMapId2.getValue();
         stream = MNodeService.getInstance(request).query(session, "solr", query);
         resultStr = IOUtils.toString(stream, "UTF-8");
+        account = 0;
+        while ( (resultStr == null || !resultStr.contains(resourceMapId2.getValue())) && account <= tryAcccounts) {
+            Thread.sleep(2000);
```

Comment:
same

Reply:
Can this be reduced from 2000, as a first step?

Then we can discuss better solutions later - see issue #1702